hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/CloseContainerCommandHandler.java

```diff
@@ -78,6 +77,7 @@ public void handle(SCMCommand command, OzoneContainer ozoneContainer,
         ((CloseContainerCommand)command).getProto();
     final ContainerController controller = ozoneContainer.getController();
     final long containerId = closeCommand.getContainerID();
+    LOG.debug("Processing Close Container command containerID={}", containerId);
```

Comment:
Minor nit: wrap in an if check, debug/trace logging params are evaluated even if the log level is disabled.

Reply:
Will do